src/main/java/edu/kis/powp/jobs2d/features/DrawerFeature.java

```diff
@@ -21,6 +21,7 @@ public static void setupDrawerPlugin(Application application) {
 		application.addComponentMenuElement(DrawPanelController.class, "Clear Panel", selectClearPanelOptionListener);
 
 		drawerController.initialize(application.getFreePanel());
+		MouseDrawFeature.SetMouseListener(application);
```

Comment:
This is an unnecessary interference in DrawerFeature and coupling between different "Features". You can probably move such configuration issues to test app class.